core/refineBindings.ml

```diff
@@ -349,7 +349,7 @@ module RefineTypeBindings = struct
         if rts || List.length sccs > 1 then
           let muName = gensym ~prefix:"refined_mu" () in
             ((tyName, muName) :: env, `Mu (muName, sugaredDT))
-        else (env, sugaredDT) in
+        else (env, fst sugaredDT) in
```

Comment:
This `fst` feels a bit "meh". It would probably be better to have a function like `un_position` or something to drop position annotation. I'll probably change that before a final merge.